cmd/net-state/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/storj/routes"
+	"github.com/storj/storage/boltdb"
+
+	"go.uber.org/zap"
```

Comment:
for imports, i'd like to do something like have

```
import (
  "stdlib"
  "stdlib"
  "stdlib"

  "thirdparty"
  "thirdparty"

  "storj"
  "storj"
)
```

so keep the stdlib together, thirdparty together, and our imports together. sound alright?

Reply:
I think goimports does this by default so you can just run that. Most IDEs have a setting to run it on save which is useful